code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -134,12 +134,13 @@
 	if(!picked)
 		pickVisualAppearence()
 
-/mob/living/simple_animal/drone/auto_deadmin_on_login()
-	if(!client?.holder)
-		return TRUE
-	if(CONFIG_GET(flag/auto_deadmin_silicons) || (client.prefs?.toggles & DEADMIN_POSITION_SILICON))
-		return client.holder.auto_deadmin()
-	return ..()
+///mob/living/simple_animal/drone/auto_deadmin_on_login()
```

Comment:
wywal to poprostu a nie komentujesz

Reply:
ale może się jeszcze przyda...

Reply:
jak sie przyda to dodamy to z gita, wszystko jest w historii zapisane